data/presets/emergency/defibrillator.json

```diff
@@ -4,11 +4,13 @@
         "indoor",
         "ref",
         "operator",
-        "defibrillator/location"
+        "defibrillator/location",
+        "opening_hours",
+        "access",
```

Comment:
The `access` field is generally used to specify access per type of transportation vehicle (foot, bicycles, motor etc.)
Given this [wiki statement](https://wiki.openstreetmap.org/wiki/Tag:emergency%3Ddefibrillator): _access=*, if the access to the device is restricted_

I think that access_simple in 'moreFields' is sufficient to cover this scenario.

Reply:
This field is often not set on defibrillators. I would highly recomment to put it on front, because then, it's not forgotten and more helpful. Thank you

Reply:
Just to clarify, could you please provide what is the expected tag key-value pair that would be included if the access field is populated? Could you provide link/id to osm entity that would serve as an example.

Reply:
Here's an example of the tags that would be used on a defi: https://www.openstreetmap.org/node/9285955621

Reply:
Please look into [access ](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/access.json) field and reconsider this. This field offers more complex specification of access, as mentioned above, it allows specification of access per type of transportation vehicle.
If the intent is for users to be more clearly asked to specify access=yes/no, then I suggest you move `access_simple` field from moreFields to fields.